apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: -6.4rem;
+  right: 0;
+  color: #D32E6A;
+  font-size: 8rem;
+  cursor: pointer;
+}
+
+.main-column {
+  width: calc(67% - 10px);
+  margin: 5px;
+}
+
+.side-column {
+  width: calc(33% - 10px);
+  margin: 5px;
+
+  img {
+    max-width: 100%;
+  }
+}
+
 .reactions {
+  .reactions-modal {
+    display: none;
+  }
+
+  &.open {
+    .reactions-modal {
+      display: block;
```

Comment:
Properties should be ordered position, top, left, z-index, display, width, height

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)